geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/AccumuloParams.scala

```diff
@@ -25,7 +27,7 @@ trait AccumuloRasterTableParam {
   var table: String = null
 }
 
-trait AccumuloConnectionParams extends OptionalZookeepersParam {
```

Comment:
we could move this logic outside the trait - the main takeaway we wanted from this PR is having the parameters set in the trait before we start processing. that's being accomplished by the invocation in `Runner.createCommand` - as long as something equivalent happens there i'm happy.